pkg/block/block_linux.go

```diff
@@ -331,7 +331,7 @@ func disks(ctx *context.Context, paths *linuxpath.Paths) []*Disk {
 		driveType, storageController := diskTypes(dname)
 		// TODO(jaypipes): Move this into diskTypes() once abstracting
 		// diskIsRotational for ease of unit testing
-		if !diskIsRotational(ctx, paths, dname) {
+		if !diskIsRotational(ctx, paths, dname) && driveType == DRIVE_TYPE_HDD {
```

Comment:
Hi @dudyas6 thanks so much for this contribution! Please add a little code comment right above here explaining why the additional `&& driveType == DRIVE_TYPE_HDD` is necessary (to address this changed kernel behaviour...)

Thank you! :)